packages/migrate/src/glint-utils.ts

```diff
@@ -87,3 +89,30 @@ export async function shouldUseGlint(basePath: string): Promise<boolean> {
     return GLINT_PROJECT_FILES.includes(pkgName);
   });
 }
+
+export const DummyPlugin = {
```

Comment:
This should implement Plugin

https://github.com/rehearsal-js/rehearsal-js/blob/c67e693979aa6becabfacbd65f64dafa8b1fc884/packages/service/src/plugin.ts#L5

Reply:
It does now by virtue of how the TS type system works (Plugin is a thing with a run method that returns `Promise<Array>` so the type system already sees it as a Plugin